src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/SubBsnesCore.cs

```diff
@@ -4,7 +4,11 @@
 
 namespace BizHawk.Emulation.Cores.Nintendo.BSNES
 {
-	[PortedCore(CoreNames.SubBsnes115, "")]
+	[PortedCore(
+		name: CoreNames.SubBsnes115,
+		author: "bsnes team",
```

Comment:
`port by Morilli`?

Reply:
Copied from
https://github.com/TASEmulators/BizHawk/blob/2f12b1efcc10420b48d4916197b84eb7d40cd5f2/src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesCore.cs#L15

I'll let @Morilli decide whether his work on this pair of cores constitutes authorship (I'm inclined to say so), and make that adjustment himself in a future commit.